app/src/main/java/com/example/oddhours/database/DatabaseHelper.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.oddhours.database
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+
+class DatabaseHelper(context: Context):
+    SQLiteOpenHelper(context, dbName, null, 1){
+
+    val sqlCreateTable_job = "CREATE TABLE IF NOT EXISTS $jobTable (" +
+            "$job_ID_COL_1 INTEGER PRIMARY KEY, " +
+            "$job_Name_COL_2 TEXT, " +
+            "$job_Location_COL_3 TEXT);"
+    val sqlCreateTable_hours = "CREATE TABLE IF NOT EXISTS $hoursTable (" +
+            "$shift_ID_COL_1 INTEGER PRIMARY KEY, " +
+            "$shift_Date_COL_2 TEXT, " +
+            "$job_ID_COL_3 INTEGER, " +
+            "$start_Time_COL_4 TEXT, " +
+            "$end_Time_COL_5 TEXT, " +
+            "$hours_Worked_COL_6 INTEGER, " +
```

Comment:
do you think it's better if we store hours worked as string instead of integer? it'll mostly be decimals (hours and minutes)

Reply:
yaa im good with storing it as string